playground/backend/pkg/fs_tool/fs.go

```diff
@@ -0,0 +1,36 @@
+package fs_tool
+
+import (
+	"beam.apache.org/playground/backend/pkg/api"
```

Comment:
Let's add alias pb for all protobuf imports

Reply:
Done.